src/python/WMCore/Services/PyCondor/PyCondorAPI.py

```diff
@@ -6,6 +6,8 @@
 
 from __future__ import print_function, division
 
+from builtins import str, object
+from past.builtins import basestring
```

Comment:
If possible, I would not import from past.

Reply:
This was inherited from #10057, I will take care of it in that PR. This one has been cleaned from the relics of #10057 and #10103 and is ready to be review again